key-vault.tf

```diff
@@ -1,18 +1,29 @@
-module "shared-vault" {
+module "old_shared_vault" {
```

Comment:
Renaming the vault is causing to cause it to be destroyed and then recreated empty. Is there a point in that?